packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -246,6 +247,9 @@ const getOrderForm = async (
         session.postalCode
```

Comment:
Adding the comment here because GitHub wouldn't allow me to put it above.

Shouldn't we send `incrementedAddress` to the shipping data API only when `shouldCreateNewAddress` is valid? Wouldn't that avoid the issue?